src/ext/service.rs

```diff
@@ -136,7 +154,7 @@ impl<'a> Iterator for Sources<'a> {
             self.libs.next().map(|(container_path, name)| {
                 match self.sources.find_by_lib_key(&name) {
                     None => Err(ErrorKind::UnknownLibKey(name).into()),
-                    Some(source) => Ok((container_path, source)),
+                    Some(source) => Ok((container_path, None, source)),
```

Comment:
No subdirs on libs yet.

Reply:
Let's just error if somebody tries to do this.